### PR TITLE
feat(XLM): set up send xlm goal

### PR DESCRIPTION
**Pull request by plondon**
## Description (optional)
Part 1 of https://blockchain.atlassian.net/browse/FWLT-20

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.js [hunk 1]
```diff
@@ -86,6 +87,17 @@ export default ({ api }) => {
     yield put(actions.router.push('/login'))
   }
 
+  const defineSendXlmGoal = function * (pathname, search) {
+    // /#/open/xlm?address={address}&amount={amount}
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.js]: No amount yet

Comment (blockdylanb) [reply packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.js]: I see memo in the code so should the format be: `/#/open/xlm?address={address}&amount={amount}&memo={memo}`?

Comment (blockdylanb) [reply packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.js]: nvm, i see it in the code right below this comment.

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.js]: You're right, dont think I included memo

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.js]: That format should work and I can make the appropriate change

**Pull request merged**